### DISCUSSION TITLE
Trigger synchronisation of file system provider from Home Assistant

### DISCUSSION THREAD

**Discussion by DarkFox** [music-assistant/support#1345] [Feature requests and ideas]
I've got a script that fetches the latest podcasts from a few feeds, and writes the url to a .m3u file for each feed, which are then are available as playlists in Music Assistant ([config, for reference](https://github.com/DarkFox/ha-config/blob/master/packages/podcasts.yaml)).

However, some of them update quite often (one of them hourly), so I'd like to be able to trigger a synchronisation from the script, so the latest podcast is always available.

**Comment by OzGav** [comment]
Suggested previously here. Please add to that discussion https://github.com/orgs/music-assistant/discussions/527
